### PR TITLE
PLT-5314 - Add Upload Document Card Component

### PR DESCRIPTION
**Pull request by akramimtiaz-byb**
https://beforeyoubid.atlassian.net/browse/PLT-5314

### CODE REVIEW

File: src/components/UploadDocumentCard/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  ${({ state, theme }) =>
+    state === 'error' &&
+    css`
+      border-color: ${theme.palette.colors.error45};
+      background-color: ${theme.palette.colors.errorL1};
+    `}
+`;
+
+const StyledLinearProgress = styled(LinearProgress)(({ theme }: { theme: Theme }) => ({
```
Comment (bailey-byb) [inline src/components/UploadDocumentCard/styles.tsx:40]: you dont need the theme typing here

**Review by bailey-byb — CHANGES_REQUESTED** [review]
few comments but overall looks good. might be worth separating each state into a unique file instead of the styles file... leave it with to you to decide though

**Comment by akramimtiaz-byb** [comment]
> few comments but overall looks good. might be worth separating each state into a unique file instead of the styles file... leave it with to you to decide though

- comments addressed
- components moved into separate files

**Review by ben-byb — APPROVED** [review]
🤷‍♂️

**Pull request merged**